testData/solutions/web.json

```diff
@@ -164,5 +164,131 @@
+                        "transform": {
+                            "type": "fluid.transforms.condition",
+                            "conditionPath": "http://registry\\.gpii\\.net/common/highContrastEnabled",
+                            "true": {
+                                "transform": {
+                                    "type": "fluid.transforms.valueMapper",
+                                    "inputPath": "http://registry\\.gpii\\.net/common/highContrastTheme",
+                                    "defaultInputValue": "default",
+                                    "options": {
+                                        "black-white": {
+                                            "outputValue": "bw"
+                                        },
+                                        "white-black": {
+                                            "outputValue": "wb"
+                                        },
+                                        "black-yellow": {
+                                            "outputValue": "by"
+                                        },
+                                        "yellow-black": {
+                                            "outputValue": "yb"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "signLanguageEnabled": "http://registry\\.gpii\\.net/common/signLanguageEnabled",
+                    "signLanguage": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/signLanguage",
+                            "options": {
+                                "ase": {
+                                    "outputValue": "ase"
+                                },
+                                "gsg": {
+                                    "outputValue": "gsg"
+                                },
+                                "ils": {
+                                    "outputValue": "ils"
+                                }
+                            }
+                        }
+                    },
+                    "interpreterType": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/signLanguageInterpreterType",
+                            "options": {
+                                "avatar": {
+                                    "outputValue": "avatar"
+                                },
+                                "human": {
+                                    "outputValue": "human"
+                                }
+                            }
+                        }
+                    },
```

Comment:
I was thinking about adding the line 
    "defaultInputValue": "avatar",
to the valueMapper, but then I noticed that this would output the settings "interpreterType" even when there is no common term "signLanguageInterpreterType" in the NP set. (See the discussion about the valueMapper for sign language, above).